cmds/modules/noded/main.go

```diff
@@ -113,6 +117,51 @@ func action(cli *cli.Context) error {
 		return nil
 	}
 
+	z := zinit.Default()
+	// monitor ntp
+	go func() {
```

Comment:
- Why this has to be in `noded`
- Why this has to be in `main` like this. That's a LOT of logic in main function. If we keep doing this for each small monitoring we need to do then main will just become a soup of un-releated pieces of code

Reply:
Please instead implement it as a `healthcheck`  https://github.com/threefoldtech/zos/blob/main/pkg/perf/healthcheck/healthcheck.go#L25

- This will allow u to run periodically
- You have time to take an action
- The state if the clock skew will be also collectable by users